curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -190,18 +203,32 @@ CURVEFS_ERROR FsCacheManager::FsSync(bool force) {
                 continue;
             } else {
                 VLOG(9) << "FileCacheManagerPtr count:"
-                        << iter1->second.use_count();
+                        << iter1->second.use_count()
+                        << ", inodeId:" << iter1->first;
                 // tmp and fileCacheManagerMap_ has this FileCacheManagerPtr, so
                 // count is 2 if count more than 2, this mean someone thread has
                 // this FileCacheManagerPtr
                 if ((iter1->second->IsEmpty()) &&
-                    (2 == iter1->second.use_count())) {
+                    (iter1->second.use_count() <= 2)) {
                     VLOG(9) << "Release FileCacheManager, inode id: "
                             << iter1->second->GetInodeId();
                     fileCacheManagerMap_.erase(iter1);
                     g_s3MultiManagerMetric->fileManagerNum << -1;
                 }
             }
+        } else if (ret == CURVEFS_ERROR::NOTEXIST) {
```

Comment:
Why can it be judged later here?

Reply:
file maybe delete，get indoe return CURVEFS_ERROR::NOTEXIST